container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -1926,6 +2394,77 @@ func (manager *KubernetesManager) HasComputeNodes(ctx context.Context) (bool, er
 	return len(nodes.Items) != 0, nil
 }
 
+func (manager *KubernetesManager) AddLabelsToNode(ctx context.Context, nodeName string, labels map[string]string) error {
+	nodeClient := manager.kubernetesClientSet.CoreV1().Nodes()
+
+	node, err := nodeClient.Get(ctx, nodeName, globalGetOptions)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while trying to get node '%v'. Ensure node with name '%v' exists in cluster.", nodeName, nodeName)
+	}
+
+	// add node selectors to existing labels
+	for k, v := range labels {
```

Comment:
This isn't a node selector, right? Just labels for the node (and how those are used is handled elsewhere but not in this function)

Reply:
correct